bait-files/WEB-INF/appengine-generated/datastore-indexes-auto.xml

```diff
@@ -1,4 +1,4 @@
-<!-- Indices written at Tue, 14 Jul 2020 19:16:49 UTC -->
+<!-- Indices written at Tue, 14 Jul 2020 22:39:27 UTC -->
 
 <datastore-indexes/>
 
```

Comment:
This file as well as local_db.bin don't seem to be necessary to the GitHub repository.  It looks like they are autogenerated any time that anybody runs the code on their own computer.  I think it should be safe to set your repository to never commit anything in the appengine-generated folder.

To do that, you just need to modify the file called .gitignore in the root directory.  I think adding the following line to the file should work:
bait-files/WEB-INF/appengine-generated/

If it does work, you should see that these two files should no longer be in the commit.

Reply:
Removed